.travis.yml

```diff
@@ -0,0 +1,65 @@
+language: c
+sudo: required
```

Comment:
If sudo usage is detected it is automatically set to yes and not build in the container-based Travis infrastructure (which is faster): http://docs.travis-ci.com/user/workers/container-based-infrastructure/